src/py42/clients/alertrules.py

```diff
@@ -56,7 +60,11 @@ def add_user(self, rule_id, user_id):
             raise
 
     def remove_user(self, rule_id, user_id):
-        """Update alert rule criteria to remove a user and all its aliases from a rule.
+        """Update alert rule criteria to remove a user and all its aliases from an alert rule. A rule's
+        user list can either be inclusive (only the users on the list will be alerted) or exclusive
+        (everyone will be alerted on except users on the list). This method will include or 
```

Comment:
probably nitpicky but maybe "include or exclude **users** based on the rule configuration" ?